app/src/main/java/com/fstyle/androidtrainning/screen/login/LoginPresenter.java

```diff
@@ -0,0 +1,84 @@
+package com.fstyle.androidtrainning.screen.login;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.util.Log;
+import com.facebook.AccessToken;
+import com.facebook.CallbackManager;
+import com.facebook.FacebookCallback;
+import com.facebook.FacebookException;
+import com.facebook.GraphRequest;
+import com.facebook.GraphResponse;
+import com.facebook.login.LoginManager;
+import com.facebook.login.LoginResult;
+import java.util.Arrays;
+import org.json.JSONException;
```

Comment:
ERROR: Unused import: org.json.JSONException. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports